docs/_docs/diagnostics.md

```diff
@@ -356,33 +456,134 @@ tuning_results = pd.DataFrame(all_params)
 tuning_results['rmse'] = rmses
 print(tuning_results)
 ```
-        changepoint_prior_scale  seasonality_prior_scale      rmse
-    0                     0.001                     0.01  0.757694
-    1                     0.001                     0.10  0.743399
-    2                     0.001                     1.00  0.753387
-    3                     0.001                    10.00  0.762890
-    4                     0.010                     0.01  0.542315
-    5                     0.010                     0.10  0.535546
-    6                     0.010                     1.00  0.527008
-    7                     0.010                    10.00  0.541544
-    8                     0.100                     0.01  0.524835
-    9                     0.100                     0.10  0.516061
-    10                    0.100                     1.00  0.521406
-    11                    0.100                    10.00  0.518580
-    12                    0.500                     0.01  0.532140
-    13                    0.500                     0.10  0.524668
-    14                    0.500                     1.00  0.521130
-    15                    0.500                    10.00  0.522980
+    19:39:23 - cmdstanpy - INFO - Chain [1] start processing
```

Comment:
Could you manually get rid of this diff pls, I assume it isn't supposed to change.